firebase_admin/auth.py

```diff
@@ -446,6 +517,36 @@ def delete_user(uid, app=None):
     user_manager.delete_user(uid)
 
 
+def delete_users(uids, app=None):
+    """Deletes the users specified by the given identifiers.
+
+    Deleting a non-existing user won't generate an error. (i.e. this method is
+    idempotent.) Non-existing users are considered to be successfully
+    deleted, and will therefore be counted in the
+    `DeleteUserResult.success_count` value.
```

Comment:
Suggest:

"Deleting a non-existing user does not generate an error (the method is idempotent). Non-existing users are considered to be successfully deleted and are therefore included in the `DeleteUserResult.success_count` value.

Reply:
Done.